php/PhpJsonApiSamples/ClientSideEncryption.php

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<!--
+To change this license header, choose License Headers in Project Properties.
+To change this template file, choose Tools | Templates
+and open the template in the editor.
+-->
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Client side Encryption</title>
+         <!-- This is the PayTrace End-to-End Encryption library: -->
+        <script src="https://rmapi.paytrace.com/assets/e2ee/paytrace-e2ee.js">
```

Comment:
Not sure if you changed in later commits but probably should use another URL for the script. Probably UAT